app/components/content_card.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Image from 'next/image';
+
+interface contentCardProps {
+    type?: contentCardType,
+    title: string,
+    text: string,
+    className?: string,
+    delimiter?: string,
+    path?: string,
+    xPosition?: string,
+    yPosition?: string,
+}
+
+enum contentCardType {
+    IMAGE = 'image',
+    TEXT = 'text',
+    LIST = 'list',
+    GENERIC = 'generic',
```

Comment:
nit: I prefer PascalCase for enum fields.

```suggestion
    Image = 'image',
    Text = 'text',
    List = 'list',
    Generic = 'generic',
```

Reply:
Great.